packages/fxa-settings/src/components/UnitRowWithAvatar/index.stories.tsx

```diff
@@ -1,22 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { LocationProvider } from '@reach/router';
-import { MockedCache } from '../../models/_mocks';
-import { UnitRowWithAvatar } from '.';
-
-storiesOf('Components|UnitRowWithAvatar', module)
-  .addDecorator((getStory) => <LocationProvider>{getStory()}</LocationProvider>)
-  .add('with default avatar', () => (
```

Comment:
Thank you for the tests 👏 

One other thing here, sorry I forgot to mention this with my first review - we use Storybook to keep track of component states here, you can pull it up by running `yarn storybook` in `fxa-settings`. `UnitRowWithAvatar` had “with default avatar” and “with non-default avatar” states. Will you add these two stories to the existing `UnitRow` stories? 🙂

Reply:
Done!